src/graphql/resolvers/mutations/adminSessionUpdate.js

```diff
@@ -0,0 +1,197 @@
+import debug from 'debug';
+
+import sessionStore from '../../../dataSources/cloudFirestore/session';
+import memberStore from '../../../dataSources/cloudFirestore/member';
+
+const dlog = debug('that:api:sessions:mutation:AdminSessionUpdate');
+
+async function updateSession(sessionId, user, session, firestore) {
+  const [updatedSession, userResults] = await Promise.all([
+    sessionStore(firestore).adminUpdate({
+      sessionId,
+      session,
+    }),
+
+    memberStore(firestore).find(user.sub),
+  ]);
+
+  return { updatedSession, userResults };
+}
+
+function sendUserEvent(
+  originalSession,
+  updatedSession,
+  userResults,
+  userEvents,
+) {
+  dlog(
+    'sendUserEvent original status: %s, updated status: %s',
+    originalSession.status,
+    updatedSession.status,
+  );
+  let eventTitle = '';
+
+  if (
+    originalSession.status === 'DRAFT' &&
+    updatedSession.status === 'SUBMITTED'
+  ) {
+    eventTitle = 'newSessionCreatedByAdmin';
```

Comment:
here you're changing the eventName that we're throwing the message against. but that isn't a valid event. Look at the /events/user to see the events we publish.

Reply:
also for this, and others. are we planning to email the end user we assigned  the session too?

Reply:
@csell5 I am thinking we should add the new event name so we know the difference, do you agree?

I didn't think through if we were going to email them. Thinking now is yes, we should email them.

Reply:
I'm not sure adding a new event name solves anything unless we wanted to change how/what we're sending for that event.

If we just want to "know" that could be done at the resolver who emitted the event in the first place.

Reply:
@csell5 My thinking is to know it was an admin mutation over a regular mutation. I suppose there is a better way to handle that. What is the event name `newSessionCreated`?

Reply:
@csell5 replaced with defined event name. tested and email is sent on create